presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/LogsHandler.java

```diff
@@ -245,4 +293,17 @@ else if (nullEnd) {
             return s -> level.name().equals(s);
         }
     }
+
+    private static Response internalError(
```

Comment:
That would cause the log entries to list the utility class as their source, which wouldn't be very useful. Though that could be solved in a few ways...

I'll look into better solutions later today, if I have time.